flexmeasures/data/scripts/cli_tasks/db_pop.py

```diff
@@ -321,3 +322,62 @@ def create_power_forecasts(
         end_of_roll=pd.Timestamp(to_date).tz_localize(timezone)
         - timedelta(hours=horizon_hours),
     )
+
+
+@app.cli.command()
+def db_dump():
+    """Create a database dump of the database used by the app."""
+    db_uri = app.config.get("SQLALCHEMY_DATABASE_URI")
+    db_host_and_db_name = db_uri.split("@")[-1]
+    click.echo(f"Backing up {db_host_and_db_name} database")
+    db_name = db_host_and_db_name.split("/")[-1]
+    time_of_saving = datetime.now().strftime("%F-%H%M")
+    dump_filename = f"pgbackup_{db_name}_{time_of_saving}.dump"
+    command_for_dumping = f"pg_dump --no-privileges --no-owner --data-only --format=c --file={dump_filename} {db_uri}"
+    try:
+        proc = subprocess.Popen(command_for_dumping, shell=True)  # , env={
+        # 'PGPASSWORD': DB_PASSWORD
+        # })
+        proc.wait()
+        dump_success = 1
+
+    except Exception as e:
+        dump_success = 0
+        click.echo(f"Exception happened during dump: {e}")
+    if dump_success:
```

Comment:
Is there any benefit in using `dump_success`over simply `click.echo` right away?